### PR TITLE
fixed app and relating files

### PR DESCRIPTION
**Pull request by shirleyyzhang**

### CODE REVIEW

File: src/main/java/interface_adapter/suggestlocation/SuggestedLocationsPresenter.java
Comment (teddyyang8) [inline src/main/java/interface_adapter/suggestlocation/SuggestedLocationsPresenter.java:1]: change variable name of loggedInState and loginState to correct naming with our project

**Review by yasmatopia — APPROVED** [review]
amazing work, thank you for your services

**Review by teddyyang8 — COMMENTED** [review]
change variable names

**Review by teddyyang8 — APPROVED** [review]
resolved issues

**Pull request merged**